src/toolbars/toolbar-component.js

```diff
@@ -52,6 +52,7 @@ angular.module('patternfly.toolbars').component('pfToolbar', {
 
     function viewSelected (viewId) {
       ctrl.config.viewsConfig.currentView = viewId;
+      ctrl.config.isTableView = viewId === 'tableView';
```

Comment:
Shouldn't this also be checked when first entering in the case where the original view is a tableView?

isTableView is not documented as an option in the config object.